src/common/greptimedb-telemetry/src/lib.rs

```diff
@@ -138,8 +144,27 @@ pub trait Collector {
     }
 }
 
-pub fn default_get_uuid() -> Option<String> {
-    let path = (*TELEMETRY_UUID_FILE_NAME).as_path();
+/// Print warning log about telemetry
+fn print_warning_log() {
+    info!("Attention: GreptimeDB now collects anonymous usage data to help improve its roadmap and prioritize features.");
```

Comment:
could use `warn!` log

Reply:
I think `info` is enough. The warning is annoying.

Reply:
I believe due to the `warning` word has a different context in the name of the method( not related to log level) can create a confusion. Possible alternative for the method name could be 
`print_anonymous_usage_data_disclaimer` or something like that.

Reply:
`print_anonymous_usage_data_disclaimer` is a better name. Thank u.